tools/documentation/development-processes-checklists.md

```diff
@@ -0,0 +1,116 @@
+1. If build succeeds and all tests pass, push `develop` to origin.
+
+_Only hotfix branches should be merged (to `master --> develop`) during this phase._
+
+Upon completion of the initialization phase, complete the sprint closeout procedure.
+
+### Before Development Phase Checklist
+1. Attempt to merge `master` into `develop` with `git merge master`.
+    - If master contains no unmerged hotfixes, git should reply `Already up-to-date.`.
+    - If master contains unmerged hotfixes, git will merge the changes (by FF if possible).
+1. If changes _were_ merged, push to origin.
+
+_Any feature/bugfix/hotfix branch may be merged (to the appropriate branches) during this phase._
+
+### Before Testing Phase Checklist
+At least three days prior to the end of the sprint, we will create a "release" branch that represents the state of the simulator after the current sprint's work would be merged into master. This release branch has a short lifespan, and exists to provide an end-user testing platform (through our staging app) as well as separation from the `develop` branch in case we want to delay the release to master, but allow the next sprint to begin on schedule.
+
+1. Create new branch `release/#.#.#` from `develop`.
+1. Open a pull request for `release/#.#.#` into `master`.
+    - Include a title of the version number, eg 'v5.2.0'.
+    - Include a description of 'Deploy v5.2.0'.
+1. On Heroku, change staging app to point to this new release branch.
+1. Broadcast publishing of testing app, and seek bug reports or feedback.
+1. Merge any applicable bugfix branches into `release/#.#.#`.
+    - And then on `develop`, run `git merge release/#.#.#` (results in a FF) and push.
+
+_Only bugfix branches should be merged (to `release/#.#.# --> develop`) during this phase._
+
+Upon completion of the testing phase, conduct the release procedure (outlined below).
+
+---
+
+## Development Procedures
+1. Merge all feature and bugfix branches:
+    - Use non-FF merges into `develop`, using either of the below commands:
+        - `git pull origin feature/###` (merges local version of branch)
+        - `git merge --no-ff feature/###` (merges latest version of upstream branch)
+    - Set/amend the merge commit's message to take the following form:
+        - Include summary of `Merge [feature/bugfix]/branchName (#PRNumber)`.
+        - Include description that very briefly explains the purpose of that branch.
+1. Merge all hotfix branches:
+    - Use non-FF merges into `master`, using either of the below commands:
+        - `git pull origin bugfix/###` (merges local version of branch)
+        - `git merge --no-ff bugfix/###` (merges latest version of upstream branch)
+    - Set/amend the merge commit's message to take the following form:
+        - Include summary of `Merge hotfix/branchName (#PRNumber)`.
+        - Include description that very briefly explains the purpose of that branch.
+    - Merge `master` into `develop` with `git merge master`.
```

Comment:
This merge _will_ have conflicts due to the different version numbers and changelog. Should resolve and allow the merge commit, using a message of `Merge hotfix/359 (#361) from master`.